test/tests/functional/pbs_multi_sched.py

```diff
@@ -327,30 +315,41 @@ def test_job_queue_partition(self):
     def test_multiple_partition_same_sched(self):
         """
         Test that multiple partition jobs are serviced by same scheduler
-        if the partitions are set on the scheduler
+        if the partitions are set on the scheduler and test that even a job
+        submitted to partition P1 queue can run on node set to partition P4
+        if both partitions are serviced by same scheduler.
```

Comment:
Hi @bhroam ,
Are you saying the comment does not sound right or the current scheduler behaviour ?
If the P1 and P4 and serviced by different schedulers , Then a job submitted to queue1 which is set to P1 will never run on node set to P4 . But now it can run . Because nodes are not assigned to queues , partitions are assigned . So scheduler can run all its available jobs on all the nodes available to it.

Reply:
I'm saying the behavior is wrong.  Job's should be contained in their partitions.  They shouldn't escape them just because the scheduler servicing the partition also services other partitions.  Think of the case where each partition is part of a machine owned by a particular department.  The department won't like it when jobs from another department suddenly starts running on their hardware.